### PR TITLE
Api Update for User 

### LINKED ISSUE #130: Fix Crash for User Api
Crash on Calling User Api
screenshot 2018-07-28 at 10 19 09 am

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Fixes #130

Changes:
Changed User Api and JSONObject to JsonArray

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@the-dagger @championswimmer review this pr

File: app/src/main/java/com/codingblocks/chatter/DashboardActivity.java [hunk 1]
```diff
@@ -74,9 +76,7 @@ protected void onCreate(Bundle savedInstanceState) {
                 avatarUrl.equals("")) {
             if (isNetworkAvailable()) {
                 Request request = new Request.Builder()
-                        .url("https://gitter.im/v1/user/me")
-                        .addHeader("Accept", "application/json")
-                        .addHeader("Authorization:", "Bearer " + accessToken)
+                        .url("https://api.gitter.im/v1/user?access_token=" + accessToken)
```
Comment (abishekvashok) [inline app/src/main/java/com/codingblocks/chatter/DashboardActivity.java]: Please update this btw :P

Comment (abishekvashok) [inline app/src/main/java/com/codingblocks/chatter/DashboardActivity.java]: Update this btw.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by aggarwalpulkit596** [comment]
@championswimmer bhaiya my commit was merged so will i be getting points for this pr?

**Pull request closed without merging**